modules/administration-guide/pages/configuring-the-administrative-user.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring the administrative user
+:keywords: administration-guide, admin
+:page-aliases:
+
+[id="configuring-the-administrative-user_{context}"]
+= Configuring the administrative user
+
+The user with administrative privileges has all {prod-short} system permissions.
+The default administrative user is `admin`.
+You can configure the user name receiving administrative privileges.
+
+.Prerequisites
+
+* An active `oc` session with administrative permissions to the OpenShift cluster. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+
+* `{prod-cli}`. See: xref:installing-the-chectl-management-tool.adoc[].
+
+.Procedure
+
+* Set the __<admin>__ user with administrative privileges:
++
+[source,yaml,subs="+attributes,+quotes,macros"]
+----
+spec:
+ server:
+   customCheProperties:
+     pass:[CHE_SYSTEM_SUPER__PRIVILEGED__MODE]: 'true'
```

Comment:
Another question: should the value for PRIVILEGED_MODE be a boolean or a string? Here it's a string, but IMHO a boolean would make more sense.

Reply:
Removing mention to CHE_SYSTEM_SUPER__PRIVILEGED__MODE, as setting to 'false' doesn't work.